src/font-stylesheet-timeout.js

```diff
@@ -57,6 +59,8 @@ function maybeTimeoutStyleSheets(win) {
 
   // Avoid timer dependency since this runs very early in execution.
   win.setTimeout(() => {
+    // Try again, more fonts might have loaded.
+    timeoutFontFaces(win);
```

Comment:
One question. When we switch timeoutFontFaces to `swap`, will we switch back to original fontface after the font have loaded?  If not, why do we have to timeout FontFaces in three different places?

Reply:
3 places (I should document) because fonts can be "discovered" over time through loading CSS.

This never changes one font twice. Once they have been touched once, they would be filtered on future iterations through the font face list.

Reply:
I see.